File: strings/silly_sentences.py
#Jared Lewis, Silly Sentences Python
print("\n")
print("Hello, and welcome to Silly sentance Creator!\n")

color = input("Please enter a single color: ").strip().lower()
name = input("Please enter a male name: ").strip().lower().capitalize()
vehicle = input("Please enter a single type of vehicle: ").strip().lower()
animal = input("Please enter an animal: ").strip().lower()
famous_person = input("Please enter the first name of a famous person: ").strip().lower().capitalize()

print("Here is your silly sentance: \n")
print("It was a " + color + " day when " + name + " decided to take a ride in his " + vehicle + ". Then he saw a " + animal + " chasing " + famous_person + " down the street. \n")

File: strings/silly_sentences.c
// Jared Lewis, Silly Senetences C
#include <stdio.h>


int main(void){
    printf("Hello, and welcome to Silly sentance Creator!\n");

    char color[20];
    printf("Please enter a single color: ");
    scanf("%s", color);

    char name[20];
    printf("Please enter a male name: ");
    scanf("%s", name);

    char vehicle[20];
    printf("Please enter a single type of vehicle: ");
    scanf("%s", vehicle);

    char animal[20];
    printf("Please enter an animal: ");
    scanf("%s", animal);

    char famous_person[20];
    printf("Please enter the first name of a famous person: ");
    scanf("%s", famous_person);

    printf("It was a %s day when %s decided to take a ride in his %s when he saw a %s chasing %s down the street. \n", color, name, vehicle, animal, famous_person);
    
    return 0;
}